core/src/main/java/org/geysermc/geyser/util/EntityUtils.java

```diff
@@ -227,6 +234,57 @@ public static void updateMountOffset(Entity passenger, Entity mount, boolean rid
         }
     }
 
+    public static Vector3f getMountOffset(Entity passenger, Entity mount) {
```

Comment:
Should this not be in TextDisplayEntity somewhere?

Reply:
No there isn't any method like that, only one similar method, but I don't recommend merging them into one.

Reply:
Most of the code here is a duplicate of the above updateMountOffset. Please find a way to consolidate/reuse code.

Reply:
Consolidate made in [5b6823b](https://github.com/GeyserMC/Geyser/pull/4676/commits/5b6823b5fba62409aca960d93de3df449186fb54)